spec/policies/restaurant_policy_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+
+RSpec.describe RestaurantPolicy, type: :policy do
+  subject { described_class }
+  
+  let(:user) { create(:user) }
+  let(:user_owner) { create(:user, email: 'owner@test.com', owner: true) }
+
+  shared_examples "User/owner cant/can to do" do
```

Comment:
create a separate file for examples, include that in spec_helper and add tests ad rising these examples for all policies, not only for this one